src/scripts.js

```diff
@@ -1,7 +1,81 @@
 import './styles.css';
 import apiCalls from './apiCalls';
+import Recipe from './classes/Recipe.js'
+import RecipeRepository from './classes/RecipeRepository.js'
+import sampleData from '../test/sampleData';
+const data = sampleData.sampleData;
 
+//extra
+let recipes, recipeRepository;
+const sampleRecipes = data.sampleRecipe
+const sampleIngredients = data.sampleIngredients
+recipes = sampleRecipes.map((recipe) => {
+  let newRecipe = new Recipe(recipe, sampleIngredients)
+  return newRecipe
+})
+recipeRepository = new RecipeRepository(recipes)
+// console.log(recipeRepository);
 
+// querySelectors
+const main = document.querySelector('#recipeContainer');
+const searchBar = document.querySelector('#searchBar')
+const searchButton = document.querySelector('#searchButton')
 
 
+// event listeners
+searchButton.addEventListener('click', collectUserInfo)
+
+// event handlers
+function collectUserInfo() {
+  let userSearch = searchBar.value;
+  let convertedUserSearch = convertUserInfo(userSearch)
+  let filteredRecipes = recipeRepository.filterRecipes(convertedUserSearch);
+  renderRecipes(filteredRecipes)
+  console.log(filteredRecipes);
+}
+
+function convertUserInfo(userSearch) {
+  let convertedUserSearch = {
+    type: 'name',
+    query: [`${userSearch}`]
+  }
+  return convertedUserSearch;
+}
+
+// other functions
+renderRecipes(recipeRepository.recipes);
```

Comment:
I would add this to the window loads event listener

Reply:
Ooh you're right, I will do that